<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>364</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>管理员系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>购票</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>501</width>
        <height>221</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发地</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到达地</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到达时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>剩余票数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>售价</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>出行日期：</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MonthSection</enum>
      </property>
      <property name="displayFormat">
       <string>MM/dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="currentSectionIndex">
       <number>0</number>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="date">
       <date>
        <year>2017</year>
        <month>6</month>
        <day>2</day>
       </date>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>购票</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>乘车人一卡通：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="LineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>-10</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>出发地：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>-10</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>到达地：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>订单查询</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>乘车人ID：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>501</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>汽车编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发地</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到达地</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到达时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>座位号</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>改签</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>退票</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
